build/teamcity/cockroach/nightlies/pebble_nightly_race_impl.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
```

Comment:
You can change this line to `set -euxo pipefail` while testing to get additional information about what command is failing in the script.